pkg/sql/catalog/descs/helpers_test.go

```diff
@@ -1,13 +0,0 @@
-// Copyright 2022 The Cockroach Authors.
-//
-// Use of this software is governed by the Business Source License
-// included in the file licenses/BSL.txt.
-//
-// As of the Change Date specified in that file, in accordance with
-// the Business Source License, use of this software will be governed
-// by the Apache License, Version 2.0, included in the file
-// licenses/APL.txt.
-
-package descs
-
-var HydrateCatalog = hydrateCatalog
```

Comment:
This function is now exported by `descs`. I'm open to moving it elsewhere but this is as good a package as any.